packages/frontend/src/pages/Search/SearchResult.tsx

```diff
@@ -21,7 +22,7 @@ export const SearchResult = () => {
     term: query,
     from: formatYear(years[0]),
     to: formatYear(years[1]),
-    fileTypes: checkedList.mime,
+    fileTypes: checkedList.mime.map((mimeType: string) => mimeNamesMapping[mimeType as keyof typeof mimeNamesMapping]),
```

Comment:
Curious: What does this bit do?

Reply:
checkedList.mime contains array of checkbox names which are slightly different from the mime names